l2geth/eth/api_tracer.go

```diff
@@ -518,6 +527,137 @@ func (api *PrivateDebugAPI) traceBlock(ctx context.Context, block *types.Block,
 	return results, nil
 }
 
+// TraceCall lets you trace a given eth_call. It collects the structured logs
+// created during the execution of EVM if the given transaction was added on
+// top of the provided block and returns them as a JSON object.
+func (api *PrivateDebugAPI) TraceCall(ctx context.Context, args ethapi.CallArgs, blockNrOrHash rpc.BlockNumberOrHash, config *TraceCallConfig) (interface{}, error) {
```

Comment:
The `TraceCall` is under `PrivateDebugAPI` which means it can't be accessed by external users. Is this intentional？

Reply:
Yeap that is intentional, it is suppose to be within the [debug namespace](https://geth.ethereum.org/docs/interacting-with-geth/rpc/ns-debug)